src/server/replica.cc

```diff
@@ -559,6 +559,9 @@ error_code Replica::InitiateDflySync() {
 
 error_code Replica::ConsumeRedisStream() {
   base::IoBuf io_buf(16_KB);
+  io::NullSink null_sink;  // we never reply back on the commands.
```

Comment:
extracting the context out from ParseAndExecute fixed the multi transaction executions.
As we get commands:
MULTI
some command
some command
EXEC

we must use the same connection context